docs/GUIDE.en.md

```diff
@@ -1249,6 +1250,132 @@ near account \
+- [from-keychain](#from-keychain---Get-the-public-key-stored-in-a-secure-keychain)
+- [from-legacy-keychain](#from-legacy-keychain---Get-the-public-key-stored-in-the-legacy-keychain-compatible-with-the-old-near-CLI)
+
+#### from-ledger - Get the public key stored on your Ledger Nano device
+
+To get the public key from Ledger, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-ledger --seed-phrase-hd-path 'm/44'\''/397'\''/0'\''/0'\''/1'\'''
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:FsRjjvkQZbwcBooXyuz4WMxXtxEKLJVJ6nc3CnaurdRr
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/jlZk2P1wCLJG6tpR8YeNONNBr?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/jlZk2P1wCLJG6tpR8YeNONNBr.png" width="836"/>
+</a>
+</details>
+
+#### from-seed-phrase - Get the public key with the seed phrase
+
+To get the public key from the seed phrase, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-seed-phrase 'trigger arrow grunt vendor crane safe reflect please sponsor verify club shiver' \
+        --seed-phrase-hd-path 'm/44'\''/397'\''/0'\'''
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:3fm1ctizEANiJG2CgJXx41e18BjtNTAnB4hfYSMjd4Fh
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/85kbNU8gg0i0LU1Cm9n9umM78?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/85kbNU8gg0i0LU1Cm9n9umM78.png" width="836"/>
+</a>
+</details>
+
+#### from-plaintext-private-key - Get the public key from the plaintext private key
+
+To get the public key from the plaintext private key, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-plaintext-private-key ed25519:3AoMxLat91aAdkh4vyq7MgbKepYhSiC5WzknLFbiXUKfsoCXXeuN9W6R4EpFd3TLvBms7gbafupvtvQJmBt7W24f
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:3fm1ctizEANiJG2CgJXx41e18BjtNTAnB4hfYSMjd4Fh
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/XV4rJx6VVxQaKDVoDz2l6Vudt?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/XV4rJx6VVxQaKDVoDz2l6Vudt.png" width="836"/>
+</a>
+</details>
+
+#### from-keychain - Get the public key stored in a secure keychain
+
+To get the public key from a secure keychain, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-keychain volodymyr.testnet \
+    network-config testnet
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
```

Comment:
nit: for another account will be similar to one as follows

Reply:
nit: doc can state, that command may block for unlocking keychain via gui-widget outside of cli input/output

Reply:
Done